bin/cask

```diff
@@ -236,26 +236,48 @@ def find_best_emacs():
     else:
         return 'emacs'
 
+def inside_emacs_24():
+    """Whether Cask is running inside Emacs-24.
+
+    Emacs 24 is rather inconsistent in its use of the ``$EMACS`` and
+    ``$INSIDE_EMACS`` environment variables, so we try here to test if Cask is
+    being run inside Emacs (either in a shell or with ``M-x compile``).
+
+    Return ``True`` when running in Emacs 24, ``False`` otherwise.
+
+    """
+    ## are we inside Emacs at all
+    return (b'INSIDE_EMACS' in ENVB and
+            ## M-x compile in Emacs-24 sets INSIDE_EMACS to exactly 't'
+            (ENVB.get(b'INSIDE_EMACS') == 't'
+             ## while M-x shell sets it to a string, starting with the
+             ## version number
+             or ENVB.get(b'INSIDE_EMACS').startswith('24')))
```

Comment:
@phillord I don't mean to annoy but I asked you to move the `or` up to the *previous* line, not onto the next one.  Just look how the line wraps *after* `and` in line 250.  When wrapping lines, boolean operators should go immediately *before* the line wrap.